Altis_Life.Altis/config/Config_Loadouts.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+    class PLAYERSIDE {
+        loadOut[] = {
```

Comment:
Having the loadout be a single array isn't consistent with the other configs and isn't very human readable (you have to refer to the comments to see where everything is constantly). Each array item should be an attribute of it's own.